CI `arm` show:
```
D:/b/usd/src/v24.11-79814a3dae.clean/pxr/base/arch/defines.h(49): fatal error C1189: #error:  "Unsupported architecture.  x86_64 or ARM64 required."
```

--- a/ports/usd/vcpkg.json
+++ b/ports/usd/vcpkg.json
@@ -1,10 +1,11 @@
 {
   "name": "usd",
   "version": "24.11",
+  "port-version": 1,
   "description": "Universal Scene Description (USD) is an efficient, scalable system for authoring, reading, and streaming time-sampled scene description for interchange between graphics applications.",
   "homepage": "https://github.com/PixarAnimationStudios/USD",
   "license": null,
-  "supports": "!x86 & !arm & !android",
+  "supports": "!x86 & !(arm & windows) & !android",
   "dependencies": [
     "opensubdiv",
     "tbb",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -9278,7 +9278,7 @@
     },
     "usd": {
       "baseline": "24.11",
-      "port-version": 0
+      "port-version": 1
     },
     "usearch": {
       "baseline": "2.3.2",

--- a/versions/u-/usd.json
+++ b/versions/u-/usd.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "d4f495620bffe1b0088ede7d5a6dd5c60a5c3f63",
+      "version": "24.11",
+      "port-version": 1
+    },
+    {
       "git-tree": "a96dae4b6c9e6d254b6601fb1be2c0088881a0b4",
       "version": "24.11",
       "port-version": 0

